### ISSUE TITLE
It is possible to set default action button for specific app?

### ISSUE THREAD

**Issue by MechanisM** [wagtail/wagtail#2106]
It is really boring to click a few times just to publish something.. Would be better to have an option to change default visible button, to "Publish" or something else. Or maybe change UI to have something like inline button-group with just icons and tooltips for them? 
I can change templates, but I needed more control over this, for example default action button for blog post would be "publish", for event would be "submit for moderation" etc.
I'm using screenshot from other issue to show as example:
![image](https://cloud.githubusercontent.com/assets/121781/12268238/f9d7cb6a-b95c-11e5-9c7b-a6d6c7354a5b.png)

**Comment by tomdyson** [comment]
@MechanisM we added keyboard shortcuts for preview (mod + p) and save (mod + s) in 1.3:

https://github.com/torchbox/wagtail/pull/1969

This uses the Mousetrap library: https://craig.is/killing/mice

It should straightforward to add new shortcuts, e.g. mod + shift + p for Publish. Would that help?

**Comment by MechanisM** [comment]
@tomdyson yeah! I even didn't noticed that.

**Comment by tomdyson** [comment]
The shortcuts aren't very well advertised. I think enabling mod+shift+p for Publish should just be adding

```javascript
Mousetrap.bind(['mod+shift+p'], function(e) {
    $('.action-publish').trigger('click');
    return false;
});
```

inside `initKeyboardShortcuts()` in `wagtail/wagtailadmin/static_src/wagtailadmin/js/page-editor.js`

and adding `action-publish` to the classes for the publish button in `wagtail/wagtailadmin/templates/wagtailadmin/pages/create|edit.html`

@MechanisM are you happy to implement / test / submit this?

**Comment by MechanisM** [comment]
@tomdyson Sure! I'll take a look what I can do on it!

**Comment by lucagiove** [comment]
A setting to change the default behavior would be very nice

**Comment by lb-** [comment]
Closing as keyboard shortcuts were implemented in #1969 and there is a fresh ongoing discussion about button layout on #2214

**Comment by lb-** [comment]
Correction - discussion about buttons are actually in #3269
